### DISCUSSION TITLE
PostsService.java#L96-L114

### DISCUSSION THREAD

**Discussion by KangSeongHyo** [dnd-side-project/dnd-5th-3-backend#128] [코드 리뷰]
https://github.com/dnd-side-project/dnd-5th-3-backend/blob/cb1f6c3d27c0978401e01b9eaf19f9a3634c2933/src/main/java/com/dnd5th3/dnd5th3backend/service/PostsService.java#L96-L114

- 문자열 보다는 enum으로 관리하면 좋을 것 같습니다.

**Comment by zooneon** [comment]
https://github.com/dnd-side-project/dnd-5th-3-backend/blob/cb1f6c3d27c0978401e01b9eaf19f9a3634c2933/src/main/java/com/dnd5th3/dnd5th3backend/controller/PostsController.java#L85-L105
요청 받을 때 string 말고 enum으로 받는게 나을 것 같은데 그렇게 하기도 하나요?
